daemon/main.c

```diff
@@ -1101,6 +1107,11 @@ int main(int argc, char **argv) {
         security_init();
 #endif
 
+        // --------------------------------------------------------------------
+        // This is the safest place to start the SILENCERS structure
+        set_silencers_filename();
```

Comment:
My initial though was not to create a new function called "set_silencers_filename" and I tried to put the code that is inside this function inside "health_initialize_global_silencers", but I copied and past the code there, I saw that the variables "silencers_filename" and "netdata_configured_varlib_dir" were not recognized inside the libnetdata/health directory. Of course, I could go more deep in the study and to adjust everything for we have this variables set there, on the other hand, the variable  silencers_filename is a global variable used in the web and database, and considering that in daemon/main.c we already initializing a lot of global variables like silencers_filename(See functions log_init and security_init), it makes sense to have them split and called here. What we could do is to move "health_initialize_global_silencers" to inside "set_silencers_filename".

Reply:
> and then have `health_initialize_global_silencers()` inside `health_silencers_init()`?
> 
> Global silencers is something that runs even when health is off ?
Yes it is, because the SILENCERS will be there, we won't send information for the clients, but this does not mean that we do not have silencers inside the code.

Reply:
> "health_initialize_global_silencers" to inside "set_silencers_filename".

sounds not good

> set_silencers_filename

should sets silencers filename and nothing more according name

Reply:
I would rename it to something like silencers_init to keep the pattern present in the file daemon/main.c .